ProQuest.js

```diff
@@ -230,9 +247,10 @@ function doWeb(doc, url, pdfUrl) {
 	//pdfUrl should be undefined unless we are calling doWeb from the following
 	//block, where it is set to false or an actual value
 	} else if(followLink && pdfUrl === undefined) {
+		
 		pdfUrl = false;
-		var link = ZU.xpathText(doc,
-			'//a[@class="formats_base_sprite format_abstract"]/@href');
+		//var link = ZU.xpathText(doc, '//a[@class="formats_base_sprite format_abstract"]/@href');
+		var link = ZU.xpathText(doc, '//li[@id="tab-AbstractRecord-null"]/a/@href');
 		if(!link) return;
 
 		//see if we can get the full text PDF link before we go
```

Comment:
the embedded PDF (see below) appears to have a different ID now (or maybe it just got an ID where it didn't used to have one). We should add "embedded-pdf" below as the first thing we check.

Also, we need to make sure that we're not attaching a "Preview" PDF when a "Full text" PDF is available. http://search.proquest.com/materials/docview/868188827/previewPDF/C0DA0CF51CE947CFPQ/8?accountid=12861 Seems like we can just check the URL for that.